detect_secrets/core/usage/audit.py

```diff
@@ -2,7 +2,14 @@
 
 
 def add_audit_action(parent: argparse._SubParsersAction) -> argparse.ArgumentParser:
-    parser = parent.add_parser('audit')
+    parser = parent.add_parser(
+        'audit',
+        help='Manually assesses a baseline to determine validity of secrets found.',
+        description=(
+            'Auditing a baseline allows analysts to optimize plugins for the highest '
```

Comment:
Nit: Maybe `Additionally, auditing..`, I more mean that, optimizing plugins is a little secondary compared to migrating out true secrets.